packages/sdk/src/dataProtector/protectDataObservable.ts

```diff
@@ -140,7 +140,8 @@ export const protectDataObservable = ({
             vName,
             JSON.stringify(schema),
             multiaddrBytes,
-            checksum
+            checksum,
+            txOptions
```

Comment:
txOptions specifies the `gasPrice` we want to inherit it from the `iexec` SDK instance